resources/eventing/charts/controller/templates/certmanager.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.enableNewCRDVersion }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: eventing-serving-cert
+  namespace: kyma-system
+spec:
+  dnsNames:
+    - eventing-controller-webhook-service.kyma-system.svc
```

Comment:
very stupid question: don't we need to use the `{{ include "controller.fullname" .}}` as we do in other templates in this folder ? I know we don't plan to rename the controller for now, but for consistency reasons it would better, right ?

Reply:
I would agree. We just left it for now since its a temporary implementation until [this](https://github.com/kyma-project/kyma/issues/15346) issue is resolved and will not go into production.